lib/cisco_node_utils/cmd_ref/interface_evpn_multisite.yaml

```diff
@@ -0,0 +1,12 @@
+# interface_evpn_multisite
+---
+_exclude: [ios_xr, N3k, N5k, N6k, N7k, N9k-F]
+
+_template:
+  get_command: "show running"
```

Comment:
Are there default values for these properties?  Does `show running all` display the defaults for them?

Reply:
No. They don't show up in the running-config if not configured.